### PR TITLE
Catalog

### PR DESCRIPTION
**Pull request by mdm-21**
- [DEMO LINK](https://mdm-21.github.io/layout_catalog/)
- [TEST REPORT LINK](https://mdm-21.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  margin-right: $card-margin-right;
+  margin-bottom: $card-margin-bottom;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
```
Comment (SanyaBratashchuk) [inline src/styles/blocks/card.scss:17]: BEM. Do not set external geometry directly to the BEM block

File: src/index.html [hunk 2]
```diff
@@ -1,12 +1,337 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="header__nav">
```
Comment (SanyaBratashchuk) [inline src/index.html:22]: nav should be different BEM block

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Text disappears on hover 
image

**Comment by ivanchuk-yurii** [comment]
don't close prs

**Pull request closed without merging**